app/Nova/Actions/MatrixAirfareSearch.php

```diff
@@ -0,0 +1,212 @@
+                $matrix_payload['slices'][0]['routingRet'] = 'N:DL';
+            } else {
+                $matrix_payload['slices'][0]['routing'] = 'C:DL+';
+                $matrix_payload['slices'][0]['routingRet'] = 'C:DL+';
+            }
+        }
+
+        return ActionResponse::openInNewTab(
+            'https://matrix.itasoftware.com/search?search='.urlencode(base64_encode(json_encode($matrix_payload)))
+        );
+    }
+
+    /**
+     * Get the fields available on the action.
+     *
+     * @return array<\Laravel\Nova\Fields\Field>
+     *
+     * @phan-suppress PhanTypeInvalidCallableArraySize
+     */
+    public function fields(NovaRequest $request): array
+    {
+        $resourceId = $request->resourceId ?? $request->resources;
+
+        if ($resourceId === null) {
+            return [];
+        }
+
+        $trip = Travel::whereId($resourceId)->sole();
+
+        return [
+            Date::make('Outbound Date')
+                ->default(static fn (): string => $trip->departure_date->format('Y-m-d'))
+                ->rules('required', 'date', 'after:tomorrow')
+                ->required(),
+
+            Select::make('Outbound Date Flexibility')
+                ->options(self::DATE_FLEXIBILITY_LABELS)
+                ->default(static fn (): string => '0')
+                ->rules('required')
+                ->required(),
+
+            Date::make('Return Date')
+                ->default(static fn (): string => $trip->return_date->format('Y-m-d'))
+                ->rules('required', 'date', 'after:outbound_date')
+                ->required(),
+
+            Select::make('Return Date Flexibility')
+                ->options(self::DATE_FLEXIBILITY_LABELS)
+                ->default(static fn (): string => '0')
+                ->rules('required')
+                ->required(),
+
+            Tag::make('Origin Airports', 'origin_airports', Airport::class)
+                ->default(static fn (): array => [['display' => 'ATL', 'value' => 'ATL']])
+                ->rules('required')
+                ->required(),
+
+            Tag::make('Destination Airports', 'destination_airports', Airport::class)
+                ->rules('required')
+                ->required(),
+
+            Number::make('Passenger Count')
+                ->min(1)
+                ->max(9)
+                ->rules('required', 'integer', 'min:1', 'max:9')
+                ->required(),
+
+            BooleanGroup::make('Policy Filters')
+                ->options(MatrixItineraryBusinessPolicy::POLICY_LABELS)
+                ->default(static fn (): array => $trip->airfare_policy ?? [])
+                ->rules('required', new FareClassPolicyRequiresMarketingCarrierPolicy())
+                ->required()
+                ->help(
+                    'Select policies to apply as search filters. <strong>Note that selected itineraries must still '.
+                    'meet the policy configured for the trip.</strong>'
```

Comment:
The way this doesn't override the `Travel`'s policy isn't 100% clear but I don't have a suggestion.